app/content/drill/[id]/index.js

```diff
@@ -42,14 +47,34 @@ export default function Index() {
   return (
     <PaperProvider>
       <SafeAreaView style={{ flex: 1 }} edges={["right", "top", "left"]}>
-        <Appbar.Header statusBarHeight={0} style={{ backgroundColor: "FFF" }}>
+        <Appbar.Header
+          statusBarHeight={0}
+          style={{ padding: 0, backgroundColor: "FFF" }}
+        >
           <Appbar.BackAction
             onPress={() => {
               navigation.goBack();
             }}
             color={"#F24E1E"}
           />
-          <Appbar.Content title={getCombinedDrillTitle(drillInfo)} />
+          <Appbar.Content
+            title={
+              <View>
+                <Text
```

Comment:
Make the top text bigger

Reply:
I'll fix that in a different PR as it does not pertain to this one

Reply:
Going to rework the drill List and drill description screens on my next PR to use the drillCard